src/yadmark/benchmark/external/apfel_utils.py

```diff
@@ -29,6 +29,7 @@ def load_apfel(theory, observables, pdf="ToyLH", use_external_grid=False):
     apfel.SetProcessDIS(observables.get("prDIS", "EM"))
     apfel.SetPropagatorCorrection(observables.get("PropagatorCorrection", 0))
     apfel.SetPolarizationDIS(observables.get("PolarizationDIS", 0))
+    apfel.SetPolarizedEvolution(observables.get("PolarizationDIS", 0))
```

Comment:
Attention! this is not a boolean, but a float - setting the amount of leptonic polarization (and only that) https://github.com/NNPDF/yadism/blob/46ca667626d5f98f98d9e7a01566ec1a7f3d0834/src/yadism/coefficient_functions/coupling_constants.py#L107
instead `SetPolarizedEvolution` refers to the hardon/proton side

Reply:
uhh ops, I was not aware of this. So the fix would be to set `SetPolarizedEvolution ` according to the observable name? Or would you prefer to have another key in the ocard ?

Reply:
I agree, I'd make it dependent on the observable name

Reply:
okay then we go for the first option